stages/org.osbuild.ostree.deploy.container

```diff
@@ -105,9 +107,49 @@ def ostree_container_deploy(tree, inputs, osname, target_imgref, kopts):
             extra_args.append(f'--target-imgref={target_imgref}')
 
             kargs = [f'--karg={v}' for v in kopts]
-
             ostree.cli("container", "image", "deploy",
                        *extra_args, sysroot=tree, *kargs)
+            
+            container_data = subprocess.run(["ostree", "container", "image", "metadata", f"--repo={tree}/ostree/repo", f"{image_source}"],
+					check=True,
+					capture_output=True, 
+					text=True).stdout.rstrip().strip('\"')
+            
+            container_data_config = subprocess.run(["ostree", "container", "image", "metadata", f"--repo={tree}/ostree/repo", f"{image_source}", "--config"],
+					check=True,
+					capture_output=True,
+					text=True).stdout.rstrip().strip('\"')
+            
+            container_data_json = json.loads(container_data)
+            container_data_config_json = json.loads(container_data_config)
+
+            aleph_digest = container_data_json['config']['digest']
+            aleph_ref = f"docker://{target_imgref}@{aleph_digest}"
```

Comment:
@dustymabe Does it make sense to have the digest as part of the `ref` if we're using the `target-imgref`? I'm thinking that if the container image input is different than the `target-imgref` specified, the digests might not be the same.

Reply:
In the traditional sense the `ref` is like a branch in an ostree repo that gets updated (i.e. a moving target). In that case I don't think it makes sense to have `ref` include the digest here.